### PR TITLE
Create _index.md

### PR DESCRIPTION
**Pull request by nag1201**
Submitting profile information for Nishant Agrawal.

### CODE REVIEW

File: content/authors/nag1201/_index.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ social:
  # link: <LINK TO YOUR TWITTER>
```
Comment (ccontrer) [inline content/authors/nag1201/_index.md:37]: maybe you should also comment line `36` `icon_pack: fab`?
also, leave no spaces before `#`.

Comment (ianabc) [reply content/authors/nag1201/_index.md:37]: Yep, if you don't need the twitter block at all you can just remove it, so your social block would end up looking like
```
social:
  icon: github
  icon_pack: fab
  link: https://github.com/nag1201
```

**Comment by ianabc** [comment]
I've added a default avatar picture for you, is that OK? I used [this one](https://m2pi.ca/authors/akcora/avatar.jpg)

If you would rather use a photo of yourself you can edit the file and replace it with your photo, just make sure it is called `content/home/nag1201/avatar.jpg`. When you are done add another commit and the pull request should automatically update. If you are OK with the generic picture instead leave a comment in here and I will merge it in.

**Comment by nag1201** [comment]
Yes that works for me.
Thank you.
Nishant

**Comment by ianabc** [comment]
Cool, I'll merge this now. Thank you!

**Pull request merged**